mandatory/2-function-creation.js

```diff
@@ -5,7 +5,15 @@ Write a function that:
 - removes any forward slashes (/) in the strings
 - makes the string all lowercase
 */
-function tidyUpString(strArr) {}
+function tidyUpString(strArr) {
```

Comment:
### Hey there so I just have some code refactoring tips for you.

- So I am pretty sure your **tidyUpString function** does pass all test cases, however we can refactor it a bit
- So good thing you are already looping and indexing through each each element
- So I do not know if you know that in JS we can chain methods like this below

```js
.replace("/", "").trim().toLowerCase();
```

```diff
- let trimmedStr = strArr[x].trim();
- let tidyStr = trimmedStr.replace("/", '') ;
- let lowerCase = tidyStr.toLowerCase(); 
+ strArr[x] = strArr[x].replace("/", "").trim().toLowerCase();
```
- Now all those 3 lines of code are just boiled down to just **1 single line**

### However we could still work on this code a bit more by making our lives easier on the looping side 

```js
let count = 0;
for (string of strArr){
    strArr[count] = string.replace("/", "").trim().toLowerCase();;
    count++;
};
```
[Refer to this link to check how that kind of for loop work's it is a game changer in certain cases](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of)

### Hope this was helpful 🔥💪